test/GameLogic/Level.test.ts

```diff
@@ -0,0 +1,111 @@
+import 'mocha';
+import {assert} from 'chai';
+import {TestLevelBuilder} from "./helpers/TestLevelBuilder";
+import {EntityType} from "../../src/GameLogic/Enums";
+import {Protagonist} from "../../src/GameLogic/Entities/Protagonist";
+
+describe('GameLogic.Level', () => {
+	describe('getEntitiesOfType', () => {
+		it("Return empty collection when no entity found", () => {
+			const level = TestLevelBuilder.newLevel().toLevel();
+
+			assert.isEmpty(level.getEntitiesOfType(EntityType.Protagonist));
+		});
+
+		it("Return empty collection when no matching entity found", () => {
+			const level = TestLevelBuilder.newLevel().toLevel();
+
+			level.entities.push({type: 21} as any);
+			level.entities.push({type: 17} as any);
```

Comment:
What are entities 21 and 17? Just dummy values for now?

Reply:
Yup, dummy values, added a comment